pages/product.html

```diff
@@ -8,146 +8,135 @@
-          <img src="../assets/vectors/collaborate-icon.svg" class="w-16 h-16" />
-          <h3>Collaborate with others</h3>
-          <p class="text-sm text-primary/80">
+        <div class="service__container-card">
+          <img
+            src="../assets/vectors/collaborate-icon.svg"
+            class="service__container-image"
+          />
+          <h3 class="service__container-header">Collaborate with others</h3>
+          <p class="service__container-text">
             Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
             nonumy eirmod tempor.
           </p>
         </div>
-        <div class="flex flex-col gap-3">
-          <img src="../assets/vectors/memory-card-icon.svg" class="w-16 h-16" />
-          <h3>Real time sync</h3>
-          <p class="text-sm text-primary/80">
+        <div class="service__container-card">
+          <img
+            src="../assets/vectors/memory-card-icon.svg"
+            class="service__container-image"
+          />
+          <h3 class="service__container-header">Real time sync</h3>
+          <p class="service__container-text">
             Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
             nonumy eirmod tempor.
           </p>
         </div>
-        <div class="flex flex-col gap-3">
-          <img src="../assets/vectors/nlp.svg" class="w-16 h-16" />
-          <h3>Built-in AI features</h3>
-          <p class="text-sm text-primary/80">
+        <div class="service__container-card">
+          <img
+            src="../assets/vectors/nlp.svg"
+            class="service__container-image"
+          />
+          <h3 class="service__container-header">Built-in AI features</h3>
+          <p class="service__container-text">
             Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
             nonumy eirmod tempor.
           </p>
         </div>
-        <div class="flex flex-col gap-3">
+        <div class="service__container-card">
           <img
             src="../assets/vectors/note-organization-icon.svg"
-            class="w-16 h-16"
+            class="service__container-image"
           />
-          <h3>Easy notes organization</h3>
-          <p class="text-sm text-primary/80">
+          <h3 class="service__container-header">Easy notes organization</h3>
+          <p class="service__container-text">
             Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
             nonumy eirmod tempor.
           </p>
         </div>
       </div>
     </section>
 
-    <section class="px-20 py-24 flex flex-col gap-10">
-      <div class="flex justify-between items-center relative">
-        <div class="flex flex-col gap-8 relative">
-          <h4 class="text-gradient">Marketing insights</h4>
-          <h2 class="leading-tight">
+    <section class="market">
+      <div class="market__content">
+        <div class="market__content-item market__content-item--position">
```

Comment:
`market__content-item--position` , position modifier is kinda abstract ... improved it